src/azurelinux/3.0/net9.0/crossdeps-builder/amd64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM azurelinuxpreview.azurecr.io/public/azurelinux/base/core:3.0
+FROM azurelinuxpreview.azurecr.io/public/azurelinux/base/core@sha256:506cbf09eda568226e75d72b2ef9f537bb069624d92bfd8856ad5b2b29fbfb20
```

Comment:
We should still be able to use the 3.0 image right?

Reply:
Yes, I meant to revert this.